hydromt_fiat/workflows/hazard.py

```diff
@@ -440,8 +437,7 @@ def create_risk_dataset(
     sorted_names = []
     
     for key, value in dict_rp_name.items():
-        map_ordered = maps[value].rename(str(key))
```

Comment:
Hi! Are the hazard test working with the changes? I created this some time ago, but I remebered I had to change the return period from int to str to make it work and save the return period correctly in the netcdf. This is the only component I identify might crash when testing risk analysis (multiple maps for different return periods).